### PR TITLE
v3.0.1 and v2.6.5 RNs also addresses feedback on v3.0.0 RN

### PR DESCRIPTION
**Pull request by emanic**
## Description

- v3.0.1 and v2.6.5 Release Notes, also incorporates suggestions from @lxpollitt 

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by tmjd** [comment]
If #1563 is being dropped in favor of this one, there are several things that I think should be added to this PR. Specifically the big warnings.

File: _data/versions.yml [hunk 1]
```diff
@@ -1,7 +1,192 @@
...
+    typha:
+      version: v0.6.0
+      url: https://github.com/projectcalico/typha/releases/tag/v0.6.0
+    calicoctl:
+      version: v2.0.0
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v2.0.0
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v2.0.0/calicoctl
+    calico/node:
+      version: v3.0.1
+      url: https://github.com/projectcalico/calico/releases/tag/v3.0.0
```
Comment (bcreane) [inline _data/versions.yml]: should be v3.0.1 (I think that's my mistake)

File: _data/versions.yml [hunk 2]
```diff
@@ -1,7 +1,192 @@
...
+    networking-calico:
+      version: 1.4.3
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.3
+      
 - title: v3.0.0
   note: |
     21 December 2017
+    
+    > **Important**: Due to a known issue in this release that can cause brief losses 
+    > of connectivty after upgrading from v2.6.4 when using the etcd datastore, this
```
Comment (bcreane) [inline _data/versions.yml]: connectivity

File: _data/versions.yml [hunk 3]
```diff
@@ -1,7 +1,192 @@
...
+    #### What's new
+    
+    ##### Breaking changes
+    
+    This release includes breaking changes to the data and resource model. [Review the documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) 
+    carefully before attempting an upgrade. Some highlights include: 
+    
+    - You must upgrade to Calico v2.6.5 before you can upgrade to v3.0.1.
+    
+    - Those accessing the etcd datastore directly must complete a one-time migration.
```
Comment (tmjd) [inline _data/versions.yml]: This sounds like it is for users that access the etcd datastore directly. I think this should be `Calico deployments using the etcd datastore must complete a one-time migration.` or something like that.

File: _data/versions.yml [hunk 4]
```diff
@@ -1,7 +1,192 @@
...
+    
+    - You must convert any customized manifests via `calicoctl convert` before you can
```
Comment (tmjd) [inline _data/versions.yml]: This is incorrect, this could be interpreted as Kubernetes manifests too.  Only Calico manifests, that use the Calico resource models, should be converted.

File: _data/versions.yml (hunk 1 above)
Comment (emanic) [reply _data/versions.yml]: OK

File: _data/versions.yml (hunk 2 above)
Comment (emanic) [reply _data/versions.yml]: Ooooops ok

File: _data/versions.yml (hunk 4 above)
Comment (emanic) [reply _data/versions.yml]: OK thanks, good info.

File: _data/versions.yml (hunk 3 above)
Comment (emanic) [reply _data/versions.yml]: OK

File: _data/versions.yml [hunk 5]
```diff
@@ -570,8 +906,8 @@ v2.6:
       url: https://github.com/projectcalico/calicoctl/releases/tag/v1.6.1
       download_url: https://github.com/projectcalico/calicoctl/releases/download/v1.6.1/calicoctl
     calico/node:
-      version: v2.6.2
-      url: https://github.com/projectcalico/calico/releases/tag/v2.6.2
+      version: v2.6.1
```
Comment (gunjan5) [inline _data/versions.yml]: why this change in 2.6.2?

Comment (emanic) [reply _data/versions.yml]: @bcreane ^^

Comment (bcreane) [reply _data/versions.yml]: That shouldn't change. only changes should be to v2.6.5 and v3.0.1, plus a RN-only change to v3.0.0.

File: _data/versions.yml [hunk 6]
```diff
@@ -570,8 +903,8 @@ v2.6:
...
+      url: https://github.com/projectcalico/calico/releases/tag/v2.6.1
```
Comment (bcreane) [inline _data/versions.yml]: this isn't right ... we should not have any changes to v2.6.2.

File: _data/versions.yml (hunk 5 above)
Comment (emanic) [reply _data/versions.yml]: OK this was an error
